### PR TITLE
adjust privatesend formatting and follow some best practices

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/privatesend/privatesend-server.cpp [hunk 1]
```diff
@@ -433,9 +434,9 @@ void CPrivateSendServer::ChargeFees(CConnman& connman)
     if (nState == POOL_STATE_ACCEPTING_ENTRIES) {
         for (const auto& txCollateral : vecSessionCollaterals) {
             bool fFound = false;
-            for (const auto& entry : vecEntries)
-                if (*entry.txCollateral == *txCollateral)
-                    fFound = true;
+            for (const auto& entry : vecEntries) {
+                if (*entry.txCollateral == *txCollateral) fFound = true;
```
Comment (UdjinM6) [inline src/privatesend/privatesend-server.cpp]: Actually, we could also fix this while at it
```suggestion
                if (*entry.txCollateral == *txCollateral) {
                    fFound = true;
                    break;
                }
```

File: src/privatesend/privatesend.cpp [hunk 2]
```diff
@@ -175,8 +174,7 @@ void CPrivateSendBaseManager::CheckQueue()
         if ((*it).IsExpired()) {
             LogPrint(BCLog::PRIVATESEND, "CPrivateSendBaseManager::%s -- Removing expired queue (%s)\n", __func__, (*it).ToString());
             it = vecPrivateSendQueue.erase(it);
-        } else
-            ++it;
+        } else ++it;
```
Comment (UdjinM6) [inline src/privatesend/privatesend.cpp]: would prefer
```suggestion
        } else {
            ++it;
        }
```

File: src/privatesend/privatesend.h [hunk 3]
```diff
@@ -43,7 +43,6 @@ enum PoolMessage {
     ERR_MN_LIST,
     ERR_MODE,
     ERR_NON_STANDARD_PUBKEY,
-    ERR_NOT_A_MN, // not used
```
Comment (UdjinM6) [inline src/privatesend/privatesend.h]: Can't do this, enum is going to be broken. Alternatively, you could change next line to `ERR_QUEUE_FULL = 14,` but I would prefer to keep it as is instead.

**Comment by PastaPastaPasta** [comment]
resolved comments, and used auto for iterators in privatesend.cpp

File: src/privatesend/privatesend-server.cpp [hunk 4]
```diff
@@ -702,9 +706,11 @@ bool CPrivateSendServer::AddScriptSig(const CTxIn& txinNew)
 // Check to make sure everything is signed
 bool CPrivateSendServer::IsSignaturesComplete()
 {
-    for (const auto& entry : vecEntries)
-        for (const auto& txdsin : entry.vecTxDSIn)
+    for (const auto& entry : vecEntries) {
+        for (const auto &txdsin : entry.vecTxDSIn) {
```
Comment (UdjinM6) [inline src/privatesend/privatesend-server.cpp]:
```suggestion
        for (const auto& txdsin : entry.vecTxDSIn) {
```

**Comment by codablock** [comment]
Seeing strange Travis failure which is completely unrelated to this PR...

**Pull request merged**